base-isa.md

```diff
@@ -85,17 +83,17 @@ TODO: This is a baseline, very much still floating
 
 
 1) Enables NEG and NOT to be encoded as `RSUB r, imm`.
-2) Placed here for now to ease decoding; `xx11` => do not store result.
+2) Placed here to ease decoding; `xx11` => do not store result.
 3) Designed to allow for building a larger immediate value. To reach a full 16 bit immediate, a 4th `SLO` or an additional `NOT` may be required.
-4) Primary intent is that these are used with immediate. Exact assignment of control registers is still floating. At least the the CPU status/extension/feature control registers should be present.
-5) The C and V flags are in an undefined state after execution of these instructions. Implementations may do whatever is easiest. An extension may mandate a particular behavior, with good enough reason, but must *not* mandate that the value of these flags after the operation depends on their value before the operation.
-6) These instructions do not have a 2 register mode. The corresponding bit patterns (`00 SS 11??`) for the first byte are reserved. This can easily be detected by using similar to 2)
-7) This instruction zero extends argument B as if the value read is of the size specified by the SS bits in the instruction (assuming a relevant extension is present)
-8) This instruction sign extends argument B as if the value read is of the size specified by the SS bits in the instruction (assuming a relevant extension is present)
+4) Control registers can only be accessed with immediates as a way to prevent potential security vulnerabilities.                                                                  
+5) The C and V flags are in an _unspecified_ state after execution of these instructions. Extensions _may_ mandate a particular behavior, with good enough reason, but must **NOT** mandate that the value of these flags after the operation depends on their value before the operation.
+6) These instructions do not have a 2 register mode. The corresponding bit patterns (`00 SS 11??`) for the first byte are _reserved_. This can easily be detected by using a similar method to 2).
+7) This instruction zero extends argument B as if the value read is of the size specified by the SS bits in the instruction (assuming a relevant extension is present).
+8) This instruction sign extends argument B as if the value read is of the size specified by the SS bits in the instruction (assuming a relevant extension is present).
```

Comment:
I think these can probably be combined into "These instructions are equivalent in the base ISA. Extensions which allow `SS` values other than `01` make these instructions distinguishable."

This simplifies the explanation significantly for someone who is only reading `base` while also very briefly justifying having both of them without explaining the nitty gritty of why (which requires some familiarity with the relevant extensions in order to make sense). Links to the 3 relevant extensions would also make sense.